server/app.js

```diff
@@ -45,25 +45,29 @@ app.listen(port, () => {
 });
 
 app.get("/repos", async (_, res) => {
-  const response = await octokit.paginate("GET /orgs/uclaacm/repos", {
-    per_page: 100,
-  });
+  try {
+    const response = await octokit.paginate("GET /orgs/uclaacm/repos", {
```

Comment:
For your try-catch block, you can probably just wrap the `await` request with the try catch block since that's the section that can cause the error instead of wrapping the entire function in one!